doc/changes/latest.inc

```diff
@@ -35,8 +35,12 @@ Current (0.23.dev0)
 
 .. |Judy D Zhu| replace:: **Judy D Zhu**
 
+.. |Valerii Chirkov| replace:: **Valerii Chirkov**
+
 Enhancements
 ~~~~~~~~~~~~
+- Update update footcite maxwell.py (:gh:`9043` **by new contributor** |Valerii Chirkov|_)
```

Comment:
```suggestion
- Update citations in maxwell.py (:gh:`9043` **by new contributor** |Valerii Chirkov|_)
```

Reply:
normally we wouldn't do a changelog entry for small documentation fixes, but its OK to leave it in here

Reply:
Alright, got it, thank you!